[2025-02-23T11:45:54.521+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-02-23T11:45:54.544+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: open_tax_etl.load_task manual__2025-02-23T11:45:50.359178+00:00 [queued]>
[2025-02-23T11:45:54.550+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: open_tax_etl.load_task manual__2025-02-23T11:45:50.359178+00:00 [queued]>
[2025-02-23T11:45:54.550+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2025-02-23T11:45:54.558+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): load_task> on 2025-02-23 11:45:50.359178+00:00
[2025-02-23T11:45:54.569+0000] {logging_mixin.py:188} WARNING - /home/airflow/.local/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py:62 DeprecationWarning: This process (pid=2410) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-02-23T11:45:54.564+0000] {standard_task_runner.py:90} INFO - Running: ['airflow', 'tasks', 'run', 'open_tax_etl', 'load_task', 'manual__2025-02-23T11:45:50.359178+00:00', '--job-id', '65', '--raw', '--subdir', 'DAGS_FOLDER/opentaxetl.py', '--cfg-path', '/tmp/tmpto18nsk7']
[2025-02-23T11:45:54.571+0000] {standard_task_runner.py:91} INFO - Job 65: Subtask load_task
[2025-02-23T11:45:54.570+0000] {standard_task_runner.py:64} INFO - Started process 2412 to run task
[2025-02-23T11:45:54.616+0000] {task_command.py:426} INFO - Running <TaskInstance: open_tax_etl.load_task manual__2025-02-23T11:45:50.359178+00:00 [running]> on host 2cf524a0af0b
[2025-02-23T11:45:54.899+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='open_tax_etl' AIRFLOW_CTX_TASK_ID='load_task' AIRFLOW_CTX_EXECUTION_DATE='2025-02-23T11:45:50.359178+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-02-23T11:45:50.359178+00:00'
[2025-02-23T11:45:54.900+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-02-23T11:45:54.928+0000] {base.py:84} INFO - Using connection ID 'opentax_postgres_conn' for task execution.
[2025-02-23T11:45:55.007+0000] {logging_mixin.py:188} INFO - Data Succesfully written to database
[2025-02-23T11:45:55.008+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-02-23T11:45:55.008+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-02-23T11:45:55.018+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=open_tax_etl, task_id=load_task, run_id=manual__2025-02-23T11:45:50.359178+00:00, execution_date=20250223T114550, start_date=20250223T114554, end_date=20250223T114555
[2025-02-23T11:45:55.061+0000] {local_task_job_runner.py:243} INFO - Task exited with return code 0
[2025-02-23T11:45:55.079+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-02-23T11:45:55.081+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
